zerver/webhooks/slack/doc.md

```diff
@@ -1,34 +1,70 @@
+# Zulip Slack integration
+
 Get Zulip notifications from Slack for messages on your team's
 public channels!
 
 See also the [Slack-compatible webhook](/integrations/doc/slack_incoming).
 
+{start_tabs}
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
+   Refer the [Channel Mapping section](#channel-mapping) for options and
+   directions on mapping your Slack channels to Zulip channels.
+
+1. Create a new [Slack app][slack_app_link] and open it.
+
+1. Navigate to the "OAuth & Permissions" menu, scroll down to the "Scopes"
+   section and make sure "Bot Token Scopes" includes:
+   `channels:read`, `channels:history`, `users:read`, `eomji:read`,
+   `team:read`, `users:read`, `users:read.email`.
+   > *Note: If you worry about the scopes that we're asking, refer to [Slack APIs used](#slack-apis-used) section to see the list of Slack APIs that we use.*
```

Comment:
You can wrap this line to make it around 70-75 characters per line for readability,